src/main/java/GeometricObject.java

```diff
@@ -0,0 +1,20 @@
+public abstract class GeometricObject {
+
+    private String color;
+    private int area;
+
+    public GeometricObject(int area, String color) {
+        this.area = area;
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getArea() {
```

Comment:
let's make this method abstract and we will override it inside of  subclasses

Reply:
done